yaptide/application.py

```diff
@@ -3,12 +3,14 @@
 from yaptide.routes import initialize_routes
 from yaptide.persistence.database import db
 from yaptide.persistence import models
+from flask_cors import CORS
 
 
 def create_app(config_object="yaptide.settings"):
     app = Flask(__name__.split('.')[0])
     app.config.from_object(config_object)
     db.init_app(app)
+    CORS(app)
```

Comment:
```suggestion
    CORS(app, supports_credentials=True)
```

browser won't save cookies without it